README.md

```diff
@@ -82,8 +82,36 @@ Start autocannon against the given target, options:
 * `duration`: the number of seconds to run the autocannon
 * `body`: a `Buffer` containing the body of the request
 * `method`: the http method to use, `GET` is the default
-* `renderProgressBar`: A truthy value to enable the rendering of the progress bar. false by default
-* `progressBarString`: A `string` defining the format of the progress display output. Must be valid input for the [progress bar module](http://npm.im/progress).
+
+**Returns** an instance/event emitter for tracking progress, etc.
+
```

Comment:
I would like to merge both of my open PR's and then do docs on the events emitted by the instance, as the 'response' event needs to be explained, and the client object which it passes to the cb: https://github.com/mcollina/autocannon/pull/14/files#diff-de78b50723ff596d7bd35665f36a6cbfR60